add example of parameter equivalance, using a URN parameter (#32)
core-wg/cf-reg-update: draft-ietf-core-cf-reg-update.md

If accepted, this request would duplicate an entry with (hypothetical)
Content-Format ID 64900 where the "Content Coding" field is left empty.

| Content Type | Content Coding | ID |
|--|--|--|
| application/my | - | 64900 |
| application/my | identity | 64999 |
{: align="left" title="Attempt at Registering an Equivalent Logical Entry with a Different Content-Format ID (2)"}

## Duplicate Entry with Equivalent Parameter

> The registrant requests an FCFS Content-Format ID for a Content Type with a parameter that has a value that looks different (based on the string) from a previously registered Content-Format that includes this parameter also.

Reviewer: split long line

```suggestion
The registrant requests an FCFS Content-Format ID for a media type that includes a parameter.
The value of this parameter appears distinct from that of a previously registered Content-Format that also includes this parameter.
```

Author: ok!

However, the semantics of the parameter value are identical to the existing registration.

> In this example, the `eat_profile` parameter value (which can be any URI) is set as a Uniform Resource Name (URN) {{RFC8141}}.

Reviewer: The URN RFC is not (yet) in the references.
We could:
```suggestion
In this example, the `eat_profile` parameter value (which can be any URI) is set as a Uniform Resource Name (URN) {{?RFC8141}}.
```
or add it to the informative refs and use it here.

> And for URNs, the Namespace Identifier (`foo` in the example) is defined as case insensitive.
> Therefore the two registrations are semantically identical.

Reviewer: join short lines :-)

```suggestion
Since for URNs, the Namespace Identifier (`foo` in the example) is defined as case insensitive, the two registrations are semantically identical.
```
